build.gradle

```diff
@@ -8,6 +8,13 @@
 
 buildscript {
     repositories {
+        maven{ url 'https://maven.aliyun.com/nexus/content/repositories/central/'}
```

Comment:
Do we really need all of these?

Reply:
You can import dependency quickly after import all of those.

Reply:
I understand that these are local CDNs for faster download, however we do not need all of these. If you really want a proper CDN then you should simply use `https://maven.aliyun.com/repository/gradle-plugin`, as that is CDN for the one we currently use (`https://plugins.gradle.org/m2/`).

Also Google ones contain just Android SDK stuff, we do not need those.